src/BloomExe/BloomExe.csproj

```diff
@@ -115,6 +115,10 @@
       <HintPath>..\..\packages\MarkdownDeep.NET.1.5\lib\.NetFramework 3.5\MarkdownDeep.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="MarkdownSharp" Condition="'$(OS)'!='Windows_NT'">
```

Comment:
Looks like you can put in xml comments: http://stackoverflow.com/a/11814977/723299